core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -19,32 +19,97 @@ package json
 
 import scala.language.implicitConversions
 
+/**
+ * This object contains the abstract snytax tree (or AST) for working with JSON objects in lift-json.
+ *
+ * The purpose of the JsonAST is to represent and manipulate JSON by leveraging Scala language features like types,
```

Comment:
Seems like “purpose of the JSON AST” would make more sense here, since you're referring to the concept rather than the class/singleton?